fr/releases/centreon-core.md

```diff
@@ -5,562 +5,110 @@ title: Centreon Core
+> Comportements connus:
+>
+>   - Si Engine sur un Poller ou un Remote Server n'est pas mis à jour en
+>     20.10, les fichiers de configuration copiés depuis un Central à jour
+>     par Gorgone utilisant une communication ZMQ ne pourront pas être
+>     lu par le processus Engine, avec l'erreur suivante dans les logs:
+>
+>     ```text
+>     Error: Parsing of global configuration failed: Can't open file '/etc/centreon-engine/centengine.cfg'
+>     ```
+>
+>     Cela est dû a des droits plus strict sur ces fichiers.
+>
+>     Comme toujours, nous **recommandons fortement** de mettre à jour
+>     tous les composants en adéquation avec la version du Central.
+>
+>     Cependant, le temps de la mise à jour, il est possible d'ajouter
+>     manuellement l'utilisateur *centreon-engine* au groupe
+>     *centreon-gorgone* avec la commande suivante :
+>
+>     ```shell
+>     usermod -a -G centreon-gorgone centreon-engine
+>     ```
 
-#### Enhancements
+### Bugfixes
 
-- Add Anomaly Detection support
-- [gRPC] Add gRPC interface
-- [gRPC] Add statistics
-- [Config] Add POLLERID & POLLERNAME macro
+- Contient tous les correctifs jusqu'à la version 20.04.7
 
 ## Centreon Broker release notes
 
-### 20.04.8
-
-#### Bug fixes
-
-- The UTF-8 chek/encoding is moved from engine to cbmod. This is easier to update for Centreon users.
-  Also, the check algorithm is fixed. Some strings could be considered as UTF-8 strings whereas they were not.
-
-### 20.04.7
+### 20.10.0
 
-#### Enhancements
-
-- Contention in conflict manager when restarting Engine with lot of
-  configuration: this version settles bases for future improvements
-
-### 20.04.6
-
-#### Bug fixes
-
-- Crash with segfault possible during a database server restart
-- BAM module could never recover after a database server restart
-
-### 20.04.5
-
-> Known issues:
+> Comportements connus:
 >
-> - When updating to 20.04.5, some badly encoded characters might block SQL
->   Broker events processing on the Central server. We strongly recommand to
->   update all the Engines to 20.04.3 version while updating to this Broker
->   version.
+>   - Si Broker sur un Poller ou un Remote Server n'est pas mis à jour en
+>     20.10 (ou utilisant une version antérieur à 20.04.9), la
+>     communication entre ces Pollers ou Remote Servers et un Central
+>     à jour peut ne pas fonctionner.
 >
->   If you experience this behaviour, we recommand to downgrade Broker on the
->   Central server using this command:
+>     Comme toujours, nous **recommandons fortement** de mettre à jour
+>     tous les composants en adéquation avec la version du Central.
 >
->   ```shell
->   yum downgrade centreon-broker-20.04.4 centreon-broker-cbd-20.04.4 centreon-broker-storage-20.04.4 centreon-broker-cbmod-20.04.4 centreon-broker-core-20.04.4
->   ```
+>     Cependant, le temps de la mise à jour, la communication peut être
+>     maintenu en s'assurant que les configurations des Brokers respectent
```

Comment:
```suggestion
>     maintenue en s'assurant que les configurations de Broker respectent
```

Reply:
There is a Broker on both Central and Poller or Remote Server, either a dedicated service or a Engine's module.